programs/fileio.c

```diff
@@ -1392,12 +1388,15 @@ int FIO_compressFilename(FIO_prefs_t* const prefs,
 static const char*
 FIO_determineCompressedName(const char* srcFileName, const char* suffix)
 {
+    
     static size_t dfnbCapacity = 0;
     static char* dstFileNameBuffer = NULL;   /* using static allocation : this function cannot be multi-threaded */
-
     size_t const sfnSize = strlen(srcFileName);
     size_t const suffixSize = strlen(suffix);
-
+    if (srcFileName == NULL) {
+        DISPLAYLEVEL(1, "ERROR: Invalid path for --output-dir-mirrored: needs to be downstream from current directory\n");
```

Comment:
Why? I thought we were going to require that all **inputs** be below the cwd, but not the output.

Reply:
Yeah that also makes sense - I'll switch it over.